graphics/gpick/Portfile

```diff
@@ -2,12 +2,14 @@
 
 PortSystem              1.0
 PortGroup               boost 1.0
+PortGroup               cmake 1.1
+PortGroup               github 1.0
+PortGroup               lua 1.0
```

Comment:
I'm not familiar with the lua portgroup. Which lua version dependency will be added? Will it correctly tell the build system to use that lua version? gpick's CMakeLists.txt looks for lua 5.4 first, then 5.3, then 5.2; therefore, in my update, I was having the port use lua 5.4.

Reply:
subport lua installs to `${prefix}/{include,lib}`
subport lua5x installs to `${prefix}/libexec/lua5x/{include,lib}`
Unless `lua.version ...` is specified, plain lua, whose version changes like any other port, is used.

The CMakefile file eventually looks for [lua-c++.pc](https://github.com/thezbyg/gpick/blob/dd27232a4dd08cf6271ecc2a7e96da25f8071ed5/CMakeLists.txt#L33) (no version).
The command `lua.finds_versioned  yes`  shows that the versioned pkgconfig files will be found first, so add a build conflict.

I have no strong feelings about this.
I have just tried to use plain lua unless there was a reason to use the versioned version.

We can easily replace `lua.finds_versioned yes` with `lua.version 5.4` to force version 5.4.

Reply:
I would consider the addition of build conflicts to be undesirable, and the use of the latest compatible version of lua to be desirable, so I'd say let's use 5.4.

Reply:
I update the PR with `lua.version  5.4`.